src/assets/scripts/client/App.js

```diff
@@ -145,8 +141,6 @@ export default class App {
      * @method enable
      */
     enable() {
-        console.info(`openScope Air Traffic Control Simulator, Version v${this.prop.version}`);
```

Comment:
How would we direct a user posting about a bug with the app frozen to determine the version they're running? To go to `Sources`, and open up index.html?